CMakeLists.txt

```diff
@@ -204,6 +207,8 @@ endif()
 if(NOT TARGET_DOMAINS OR TARGET_DOMAINS STREQUAL "None")
   # Set to all by default
   set(TARGET_DOMAINS ${DOMAINS_LIST})
+  # Remove sparse_blas from the default until it is supported by MKLCPU and MKLGPU backends
+  list(REMOVE_ITEM TARGET_DOMAINS "sparse_blas")
```

Comment:
I wanted to prevent users to build spBLAS example by default because currently the examples throw unsupported. With your suggestion users running `ctest` with only the `MKLCPU` backend would get failures because of this.
I think you are right I could make this change in the [next PR](https://github.com/oneapi-src/oneMKL/pull/387) which adds support for gemv and make the examples pass.
This relates on your comment to discuss how we should split the [PRs](https://github.com/oneapi-src/oneMKL/pull/374#discussion_r1337870961).